<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::ModelInspectorWidget</class>
 <widget class="QWidget" name="GammaRay::ModelInspectorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <item>
         <widget class="QSplitter" name="splitter_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="layoutWidget_3">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="KFilterProxySearchLine" name="modelSearchLine"/>
            </item>
            <item>
             <widget class="QTreeView" name="modelView">
              <property name="uniformRowHeights">
               <bool>true</bool>
              </property>
              <property name="allColumnsShowFocus">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget_4">
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Model Content:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTreeView" name="modelContentView"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget_5">
           <layout class="QVBoxLayout" name="verticalLayout_13">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Cell Content:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTreeView" name="modelCellView">
              <property name="rootIsDecorated">
               <bool>false</bool>
              </property>
              <property name="uniformRowHeights">
               <bool>true</bool>
              </property>
              <property name="allColumnsShowFocus">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
 </widget>
 <resources/>
 <connections/>
 <customwidgets>
  <customwidget>
   <class>KFilterProxySearchLine</class>
   <extends>QWidget</extends>
   <header>kde/kfilterproxysearchline.h</header>
  </customwidget>
 </customwidgets>
</ui>
